pkg/schedule/affinity/manager.go

```diff
@@ -79,7 +79,7 @@ type Manager struct {
 // NewManager creates a new affinity Manager.
 func NewManager(ctx context.Context, storage endpoint.AffinityStorage, storeSetInformer core.StoreSetInformer, conf config.SharedConfigProvider, regionLabeler *labeler.RegionLabeler) (*Manager, error) {
```

Comment:
Can we review this before the New Manager?

Reply:
In code order, it is 
```
	c.regionLabeler, err = labeler.NewRegionLabeler(c.ctx, c.storage, regionLabelGCInterval)
	if err != nil {
		return err
	}

	// create affinity manager with region labeler for key range validation and rebuild
	c.affinityManager, err = affinity.NewManager(c.ctx, c.storage, c, c.GetOpts(), c.regionLabeler)
	if err != nil {
		return err
	}
```
So it only defensive programming